Looms.PoS/Looms.PoS.Application/Services/TokenService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Looms.PoS.Application.Constants;
+using Looms.PoS.Application.Interfaces.ModelsResolvers;
+using Looms.PoS.Application.Interfaces.Services;
+using Looms.PoS.Application.Models.Responses.Auth;
+using Looms.PoS.Domain.Daos;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+
+namespace Looms.PoS.Application.Services;
+
+public class TokenService : ITokenService
+{
+    private readonly IConfiguration _configuration;
+    private readonly IAuthModelsResolver _authModelsResolver;
+
+    public TokenService(IConfiguration configuration, IAuthModelsResolver authModelsResolver)
+    {
+        _configuration = configuration;
+        _authModelsResolver = authModelsResolver;
+    }
+
+    public LoginResponse CreateToken(UserDao userDao)
+    {
+        var expires = DateTime.UtcNow.AddHours(TokenConstants.TokenDurationInHours);
+        var userRole = userDao.Role;
+
+        var tokenHandler = new JwtSecurityTokenHandler();
+        var key = Encoding.ASCII.GetBytes(_configuration["Jwt:Key"]!);
+        var tokenDescriptor = new SecurityTokenDescriptor
+        {
+            Subject = new ClaimsIdentity(
+            [
+                new Claim(ClaimTypes.NameIdentifier, userDao.Id.ToString()),
+                new Claim(ClaimTypes.Role, userRole.ToString()),
+                new Claim(TokenConstants.BusinessIdClaim, userDao.BusinessId.ToString())
```

Comment:
businessId can be null for super admin, also don't know if we need to add it to claim

Reply:
removed businessId claim